templates/flavors/k3s/secret.yaml

```diff
@@ -39,11 +39,15 @@ stringData:
      name: ccm-linode
     spec:
      targetNamespace: kube-system
-     version: ${LINODE_CCM_VERSION:=v0.3.24}
+     version: ${LINODE_CCM_VERSION:=v0.3.25}
```

Comment:
should we bump this to 0.4.1?

Reply:
v0.3.25 works for us, it won't work for anyone running without VPC. I updated it to v0.4.1 just to keep using the latest version.